### ISSUE TITLE
Allow more control over file split / ability to merge js files

### ISSUE THREAD

**Issue by muzidudu** [sveltejs/kit#9520]
### Describe the problem

After building too many files, can all css and js files be merged? Since the project is not large, I feel that too many requests are not more efficient than individual ones.

<img width="1492" alt="截屏2023-03-27 01 43 37" src="https://user-images.githubusercontent.com/9375220/227794340-1c59db35-2c1f-4023-a73e-ef80c450426b.png">

### Describe the proposed solution

Is it possible to add a configuration to decide whether the code needs to be split? After trying the vite configuration item build-rollupOptions-output-manualChunks, it seems to have no effect and will prompt an error\
<img width="800" alt="截屏2023-03-27 01 51 37" src="https://user-images.githubusercontent.com/9375220/227794576-1425b9bd-8c14-436a-bbda-7f5042c68ad5.png">

### Importance

would make my life easier

**Comment by rubenfiszel** [comment]
For windmill (https://github.com/windmill-labs/windmill), we are building a big single-page-app, and it would make it a lot easier if we could just have one bundle-<hash>.js to download

**Comment by benmccann** [comment]
> Since the project is not large, I feel that too many requests are not more efficient than individual ones

That doesn't seem accurate without more details. The files will be downloaded in parallel, so there's no performance hit to splitting them. There are however several performance wins to splitting the files. It means you don't need to request all the JS for the site, but only for the page you're on. It also means better caching when you update your site as only the files that have changed need to be re-requested instead of re-downloading all the JS for your whole site

**Comment by benmccann** [comment]
Closing as duplicate of https://github.com/sveltejs/kit/issues/3882
